update-yarn/create-pr.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+# The new Yarn version is passed as an argument to this script
+NEW_YARN_VERSION=$1
+GITHUB_TOKEN=$2
+GITHUB_REPOSITORY=$3
+BRANCH_NAME="yarn-update-$NEW_YARN_VERSION"
+
+# Configure git
+git config --global user.name 'github-actions'
+git config --global user.email 'github-actions@github.com'
+
+# Create a new branch for the Yarn update
+git checkout -b "$BRANCH_NAME"
```

Comment:
What happens if the branch exists (e.g. no-one has merged the PR since the branch was originally created)? Could we add `|| exit 1` here, similar to the compliance reviews?

Example: https://github.com/equinor/fusion-cpr-api/blob/62a01ba5de204bdfe5da29112bec32b61cd9946a/.github/workflows/scheduled_compliance_review.yaml#L25

Reply:
We have `set -e` at the top of the script to automatically exit whenever a command fails. In short, it will fail and we probably won't notice 😅 On the other hand, if somebody does merge the previous update, the next PR should appear within 24 hours.

Reply:
Since I'm going to make changes anyway, I'll add a check to see if the branch exists before doing anything so the failure is controlled.